lstchain/scripts/lstchain_data_create_time_calibration_file.py

```diff
@@ -77,24 +78,29 @@ def main():
     # declare the pedestal calibrator
     lst_r0 = LSTR0Corrections(pedestal_path=args.pedestal_file, config=config)
 
+    reader = event_source(input_url=path_list[0], max_events=args.max_events)
     # declare the time corrector
     timeCorr = TimeCorrectionCalculate(calib_file_path=args.output_file,
                                        config=config,
                                        subarray=reader.subarray)
 
     tel_id = timeCorr.tel_id
 
-    for i, event in enumerate(reader):
-        if event.index.event_id % 5000 == 0:
-            print(event.index.event_id)
+    for path in path_list:
+        print("--> Processing: {}".format(path))
+        reader = event_source(input_url=path, max_events=args.max_events)
+        print("--> Number of files", reader.multi_file.num_inputs())
+        for i, event in enumerate(reader):
+            if event.index.event_id % 5000 == 0:
+                print(event.index.event_id)
```

Comment:
maybe `logging` instead of `print`?
also it would be nice to have something like:
`log.info(f'File {j} out of {len(path_list)}')` 
(for that you need to change l.89 to `for i, path in enumerate(path_list)`)

Reply:
Yes, will be much better. 
Done.